src/components/ConnectedList.jsx

```diff
@@ -33,7 +34,7 @@ class ConnectedList extends Component {
   }
 
   render() {
-    const { t, connectedKonnectors, wrapper } = this.props
+    const { t, connectedKonnectors, storeUrl, wrapper } = this.props
```

Comment:
As `store` is a verb, `storeUrl` must mean `store an URL`. It may be confusing.

Reply:
*store* is also a noun though :)

Reply:
`storeAppUrl`?

Reply:
> store is also a noun though :)

I know, that is why I called it _confusing_.

The word _update_ is a noun and a verbe, but naming a variable `updateURL` is something as confusing as `storeURL`.